app/javascript/packages/analytics/package.json

```diff
@@ -2,7 +2,16 @@
   "name": "@18f/identity-analytics",
   "private": true,
   "version": "1.0.0",
+  "scripts": {
+    "postinstall": "make digital-analytics-program.js"
+  },
+  "exports": {
+    ".": "./index.ts",
+    "./click-observer-element": "./click-observer-element.ts",
+    "./digital-analytics-program": "./digital-analytics-program.js"
```

Comment:
is there any kind of basic test we can add for this resulting file? or would it require stubbing too much Google analytics?

Reply:
> is there any kind of basic test we can add for this resulting file? or would it require stubbing too much Google analytics?

Maybe a basic test that checks for syntax errors or that it can be loaded successfully? 

Could be something like the old `es5-safe` package we had (#4296).

Reply:
> is there any kind of basic test we can add for this resulting file? or would it require stubbing too much Google analytics?

I added a test to check for basic syntax error.

Originally I went a bit further in 4c4fb997ea to setup the DOM and ensure that the DAP script loaded correctly. While the test passes on its own, the DAP script does a lot of DOM manipulation side effects that I don't feel confident about trying to clean-up after. As I mentioned in extended commit description of f161089980, it'd be nice to find a way to create an isolated DOM to run DAP in, but we don't have a clear way to do that currently. In the future, maybe we can repurpose [spec_helper.js](https://github.com/18F/identity-idp/blob/main/spec/javascript/spec_helper.js) and [support/dom.js](https://github.com/18F/identity-idp/blob/main/spec/javascript/support/dom.js) to be more reusable.